frontend/src/utils/api.js

```diff
@@ -67,6 +81,28 @@ const api = {
     listTypes: () => get("/instruments/list_types"),
   },
 
+  libraries: {
+    get: libraryId => get(`/libraries/${libraryId}/`),
+    list: (options, abort) => get("/libraries", options, { abort }),
+    listExport: options => get("/libraries/list_export/", {format: "csv", ...options}),
+    summary: () => get("/libraries/summary/"),
+    template: {
+      actions: () => get(`/libraries/template_actions/`),
+      check:  (action, template) => post(`/libraries/template_check/`, form({ action, template })),
+      submit: (action, template) => post(`/libraries/template_submit/`, form({ action, template })),
+    },
+    prefill: {
+      templates: () => get(`/libraries/list_prefills/`),
+      request: (options, template) => get(`/libraries/prefill_template/`, {template: template, ...options}),
+    },
+    search: q => get("/libraries/search/", { q }),
+  },
+
+  libraryTypes: {
+    get: libraryTypeId => get(`/library-types/${libraryTypeId}/`),
+    list: (options, abort) => get("/library-types/", options, { abort }),
+  },
+
```

Comment:
Missing platforms api calls. The action reducers are still there and use api.platforms.list(). We should either keep platform API call or remove all traces of it.